protocol/src/fschacha20poly1305.rs

```diff
@@ -42,7 +42,7 @@ impl std::error::Error for Error {
 /// nonces and re-keying, providing forward secrecy within the session.
 ///
 /// FSChaCha20Poly1305 is used for message packets in BIP324.
-#[derive(Clone, Debug)]
```

Comment:
Is the drop of `Debug` here for the sake of secret hiding?

Reply:
Yea, things like the key and even he message_counter shouldn't end up in logs or what not.